### PR TITLE
[MOSIP-29035] Updated Dockerfile to fetch partner onboarder values from docker env vars

### PR DESCRIPTION
**Pull request by akilalakshmanan**

### CODE REVIEW

**Review by Mahesh-Binayak — APPROVED** [review]
Approved

File: default.sh [hunk 1]
```diff
@@ -561,4 +575,7 @@ elif [ "$MODULE" = "resident-oidc" ]; then
   LOGO_URI="https://$( printenv mosip-mimoto-host )/assets/inji-oidc.png"
   REDIRECT_URI="io.mosip.residentapp.inji:\/\/oauthredirect"
   onboard_mimoto_oidc_partner
+  echo "Updating Mimoto OIDC Partner Client ID"
+  kubectl create secret generic mimoto-oidc-partner-clientid -n mimoto --from-literal=mimoto-oidc-partner-clientid=$mpartner-default-mimotooidc-clientID --dry-run=client -o yaml | kubectl apply -f -
```
Comment (syedsalman3753) [inline default.sh:579]: namespace hardcoded, please take it as an env variable

File: default.sh [hunk 2]
```diff
@@ -545,6 +556,9 @@ elif [ "$MODULE" = "resident-oidc" ]; then
   root_cert_path="$MYDIR/certs/$PARTNER_KC_USERNAME/RootCA.pem"
   client_cert_path="$MYDIR/certs/$PARTNER_KC_USERNAME/Client.pem"
   onboard_mimoto_keybinding_partner
+  echo "Updating Mimoto Wallet Binding Partner API Key"
+  kubectl create secret generic mimoto-wallet-binding-partner-api-key -n mimoto --from-literal=mimoto-wallet-binding-partner-api-key=$mpartnerdefaultmimotokeybindingapikey --dry-run=client -o yaml | kubectl apply -f -
```
Comment (syedsalman3753) [inline default.sh:560]: namespace hardcoded, please take it as an env variable

File: default.sh [hunk 3]
```diff
@@ -519,6 +522,11 @@ elif [ "$MODULE" = "demo-oidc" ]; then
   root_cert_path="$MYDIR/certs/$PARTNER_KC_USERNAME/RootCA.pem"
   client_cert_path="$MYDIR/certs/$PARTNER_KC_USERNAME/Client.pem"
   onboard_relying_party_with_demo_oidc_client
+  echo "Updating jwk privateandpublickeypair and Mpartner Default Demo Oidc Client ID"
+  kubectl patch secret mock-relying-party-service-secrets -n esignet -p '{"data":{"client-private-key":"'$(echo -n "$privateandpublickeypair" | base64 | tr -d '\n')'"}}'
+  kubectl rollout restart deployment -n esignet mock-relying-party-service
+  kubectl -n esignet set env deployment/mock-relying-party-ui CLIENT_ID=$mpartnerdefaultdemooidcclientID
+  echo "JWK PrivatePublic Key Pair and Mpartner Default Demo Oidc Client ID updated successfully"
```
Comment (syedsalman3753) [inline default.sh:529]: namespace hardcoded, please take it as an env variable

File: default.sh [hunk 4]
```diff
@@ -504,6 +504,9 @@ elif [ "$MODULE" = "esignet" ]; then
   PARTNER_ORGANIZATION_NAME=IITB
   PARTNER_TYPE=Misp_Partner
   onboard_esignet_partner
+  echo "Updating esignet MISP_LICENSE_KEY"
+  kubectl create secret generic esignet-misp-onboarder-key -n esignet --from-literal=mosip-esignet-misp-key=$MISP_LICENSE_KEY --dry-run=client -o yaml | kubectl apply -f -
```
Comment (syedsalman3753) [inline default.sh:508]: namespace hardcoded, please take it as an env variable

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

**Review by syedsalman3753 — APPROVED** [review]

**Pull request merged**